patches/server/1034-Fix-UnsafeValues-loadAdvancement.patch

```diff
@@ -0,0 +1,29 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: LemonCaramel <admin@caramel.moe>
+Date: Sun, 24 Sep 2023 20:19:44 +0900
+Subject: [PATCH] Fix UnsafeValues#loadAdvancement
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java b/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
+index 96f6e0554baf5915dd1f5b93f3bcfe7a13393c29..eec5c5be008a8ada2a81fc3858ab8df13172e595 100644
+--- a/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
++++ b/src/main/java/org/bukkit/craftbukkit/util/CraftMagicNumbers.java
+@@ -314,7 +314,17 @@ public final class CraftMagicNumbers implements UnsafeValues {
+         JsonObject jsonobject = GsonHelper.convertToJsonObject(jsonelement, "advancement");
+         net.minecraft.advancements.Advancement nms = net.minecraft.advancements.Advancement.fromJson(jsonobject, new DeserializationContext(minecraftkey, MinecraftServer.getServer().getLootData()));
+         if (nms != null) {
+-            MinecraftServer.getServer().getAdvancements().advancements.put(minecraftkey, new AdvancementHolder(minecraftkey, nms));
++            // Paper start - Fix throw UnsupportedOperationException
++            //MinecraftServer.getServer().getAdvancements().advancements.put(minecraftkey, new AdvancementHolder(minecraftkey, nms));
++            final var mapBuilder = com.google.common.collect.ImmutableMap.<ResourceLocation, AdvancementHolder>builder();
```

Comment:
Quick, get rid of var before kenny sees this!

Reply:
ok fixed it